packages/rocketchat-livechat/livechat.js

```diff
@@ -30,15 +30,29 @@ WebApp.connectHandlers.use('/livechat', Meteor.bindEnvironment((req, res, next)
 
 	const html = `<html>
 		<head>
+		<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js" type="text/javascript"></script>
```

Comment:
As far as I understood, jQuery is only used in order to dynamically add the mobile class after loading on a mobile device.
This can also be achieved using native DOM-Events, see https://gomakethings.com/a-native-javascript-equivalent-of-jquerys-ready-method/ .
If possible, I'd like not to include a library dependency.

Reply:
I  have tryed with the DOM-Events, but there was also a problem with the Iframe.

Reply:
I agree that loading jquery should be prevented here.
@HenrikScholl Did you try the referenced code from @mrsimpson:

```
var ready = function ( fn ) {

    // Sanity check
    if ( typeof fn !== 'function' ) return;

    // If document is already loaded, run method
    if ( document.readyState === 'complete'  ) {
        return fn();
    }

    // Otherwise, wait until document is loaded
    document.addEventListener( 'DOMContentLoaded', fn, false );

};

// Example
ready(function() {
    // Do stuff...
});
```

And if we do not get around using `jquery`, it should not be loaded from `ajax.googleapis.com` for several reasons:

* CORS: serving pages over SSL should not load resource from an unsecure source
* tracking: we won't let google know our pages that are using jquery
* conflicts: this jquery version might be in conflict with the one used within the page context